tests/unit/models/gpflux/test_models.py

```diff
@@ -136,8 +136,7 @@ def test_deep_gaussian_process_optimize_with_defaults(
     y_observed = fnc_2sin_x_over_3(x_observed)
     data = x_observed, y_observed
     dataset = Dataset(*data)
-    optimizer = BatchOptimizer(tf.optimizers.Adam())
-    model = DeepGaussianProcess(two_layer_model(x_observed), optimizer)
+    model = DeepGaussianProcess(two_layer_model(x_observed))
```

Comment:
Is this change because you're using the default `fit_args` so you don't need to pass the optimizer?